### PR TITLE
Adds RDFG number in ExpandVolume API

### PR DESCRIPTION
**Pull request by delldubey**
# Description
- Updates ExpandVolume to support RDFG number
- This enables SRDF volumes to be expanded using the API

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/553|

# Checklist:

- [x] Have you run format,vet & lint checks against your submission?
- [x] Have you made sure that the code compiles?
- [x] Did you run the unit & integration tests successfully?
- [ ] Have you maintained at least 90% code coverage?
- [x] Have you commented your code, particularly in hard-to-understand areas
- [x] Have you done corresponding changes to the documentation
- [x] Did you run tests in a real Kubernetes cluster?
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration
- Unit Tests
411 scenarios (411 passed)
2826 steps (2826 passed)
PASS

### CODE REVIEW

**Review by boyamurthy — APPROVED** [review]
LGTM.

File: interface.go [hunk 1]
```diff
@@ -133,30 +133,32 @@ type Pmax interface {
 	// DeleteMaskingView deletes a masking view given a masking view id
 	DeleteMaskingView(ctx context.Context, symID string, maskingViewID string) error
 
-	// Get the list of Storage Pools
+	//GetStoragePoolList Gets the list of Storage Pools
 	GetStoragePoolList(ctx context.Context, symID string) (*types.StoragePoolList, error)
 
-	// Rename a Volume given the volumeID
+	//RenameVolume Renames a Volume given the volumeID
```
Comment (shanmydell) [inline interface.go:139]: renames - case sensitive

File: interface.go [hunk 2]
```diff
@@ -133,30 +133,32 @@ type Pmax interface {
...
 	RenameVolume(ctx context.Context, symID string, volumeID string, newName string) (*types.Volume, error)
 
-	// Add volume(s) asynchronously to a StorageGroup
+	//AddVolumesToStorageGroup Adds volume(s) asynchronously to a StorageGroup
```
Comment (shanmydell) [inline interface.go:142]: Adds sentencecase

File: interface.go [hunk 3]
```diff
@@ -133,30 +133,32 @@ type Pmax interface {
...
 
-	// Remove volume(s) synchronously from a StorageGroup
+	//RemoveVolumesFromStorageGroup Remove volume(s) synchronously from a StorageGroup
 	RemoveVolumesFromStorageGroup(ctx context.Context, symID string, storageGroupID string, force bool, volumeIDs ...string) (*types.StorageGroup, error)
 
 	// RemoveVolumesFromProtectedStorageGroup removes one or more volumes (given by their volumeIDs) from a Protected StorageGroup.
 	RemoveVolumesFromProtectedStorageGroup(ctx context.Context, symID string, storageGroupID, remoteSymID, remoteStorageGroupID string, force bool, volumeIDs ...string) (*types.StorageGroup, error)
 
-	// Initiate a job to remove storage space from the volume.
+	//InitiateDeallocationOfTracksFromVolume Initiate a job to remove storage space from the volume.
```
Comment (shanmydell) [inline interface.go:158]: Initiate sentencecase

File: interface.go [hunk 4]
```diff
@@ -133,30 +133,32 @@ type Pmax interface {
...
 	InitiateDeallocationOfTracksFromVolume(ctx context.Context, symID string, volumeID string) (*types.Job, error)
 
-	// Deletes a volume
+	//DeleteVolume Deletes a volume
```
Comment (shanmydell) [inline interface.go:161]: sentencecase

**Pull request merged**